sensirion_common.c

```diff
@@ -38,6 +38,21 @@
 #include "sensirion_common.h"
 #include "sensirion_i2c.h"
 
+uint32_t sensirion_bytes_to_uint32_t(const uint8_t* bytes) {
+    return (uint32_t)bytes[0] << 24 | (uint32_t)bytes[1] << 16 |
+           (uint32_t)bytes[2] << 8 | (uint32_t)bytes[3];
+}
+
+uint32_t sensirion_bytes_to_float(const uint8_t* bytes) {
+    union {
+        uint32_t u32_value;
+        float float32;
+    } tmp;
+
+    tmp.u32_value = sensirion_bytes_to_uint32(bytes);
```

Comment:
Huh, this gives a warning because the function name is wrong. But we should really error on warnings during CI.

Reply:
fixed (first push only fixes, the second rebases on top of master, which would error on warnings)